test/suites/ami/testdata/br_userdata_input.sh

```diff
@@ -1,4 +1,9 @@
 [settings.kubernetes]
 kube-api-qps = 30
+eviction-max-pod-grace-period = 40
```

Comment:
I think we might want to add a few unit tests to verify that these fields are getting propagated on to the node: https://github.com/aws/karpenter-provider-aws/blob/main/pkg/providers/launchtemplate/suite_test.go#L1393